workflow-templates/techdocs.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish TechDocs Site
+
+on:
+  push:
+    branches: [main]
+    # Run only when TechDocs related files are updated.
+    paths:
+      - "docs/**"
+      - "mkdocs.yml"
+
+jobs:
+  publish-techdocs-site:
+    runs-on: ubuntu-latest
+
+    # Use GitHub Organization secrets to set them for all existing and new repositories.
+    env:
+      TECHDOCS_GCS_BUCKET_NAME: ${{ secrets.TECHDOCS_GCS_BUCKET_NAME }}
+      GCS_ACCESS_KEY_ID: ${{ secrets.GCS_ACCESS_KEY_ID }}
+      GCS_SECRET_ACCESS_KEY: ${{ secrets.GCS_SECRET_ACCESS_KEY }}
```

Comment:
That would be the one Core Infra created - TECHDOCS_GCS_SECRET_ACCESS_KEY_PRODUCTION, woudln't it?